CSS/style.css

```diff
@@ -0,0 +1,108 @@
+*{
+    box-sizing: border-box;
+}
+
+/* Navigation */
+
+#header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: whitesmoke;
+}
+#header-img{
+    
+}
```

Comment:
It is considered good practice to not use an empty ruleset. Please remove this ruleset if not necessary.